hnn_core/cell.py

```diff
@@ -148,6 +148,51 @@ def gid(self, gid):
             raise RuntimeError('Global ID for this cell already assigned!')
 
 
+class Section:
+    """Section class.
+
+    Parameters
+    ----------
+    L : float
+        length of a section in microns
+    diam : float
+        diameter of a section in microns
+    cm : float
+        membrane capacitance in micro-Farads
+    Ra : float
+        axial resistivity in ohm-cm
```

Comment:
Attributes section for `mechs` and `syns`?

Reply:
sure! overall, what's your feeling? do you approve of this direction?

Reply:
Definitely approve. It may just be a product of better documentation, but I find this much more inspectable compared to a generic nested dictionary.